deployments/rialto/Front-end.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:12 as build-deps
+
+
+# install tools and dependencies
+RUN set -eux; \
+	apt-get install -y git
+
+# clone UI repo
+RUN cd /usr/src/ && git clone https://github.com/paritytech/bridge-ui.git
+WORKDIR /usr/src/bridge-ui
+RUN yarn
+ENV SUBSTRATE_PROVIDER wss://wss.rialto.brucke.link
```

Comment:
Also in my tests I've removed those, the default are already set in the UI and in the docker-compose.

Reply:
Then it explains why its broken. in Dockerfile you have to be explicit about ENV.

Reply:
What do you mean with explicit, when I tried `ENV SUBSTRATE_PROVIDER` without default, hoping it would pick up what was passed by environment, docker complained "ENV must have two arguments".

Reply:
You must always provide a default, cause it might not be set.

Reply:
I was partly wrong in my previous [comment](https://github.com/paritytech/parity-bridges-common/pull/340#issuecomment-689623338). The values were either the UI's default, or the Dockerfile's default ENV.

If I keep those ENV here, they replace anything, and don't pick-up what was set in the `docker-compose`.
```
ENV SUBSTRATE_PROVIDER aaaaaws://bridge-node-alice:9944
ENV ETHEREUM_PROVIDER aaaaahttp://poa-node-arthur:8545
```

results in the UI getting those "aaa*" values.